.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -3,12 +3,14 @@ description: Propose an enhancement for Babelfish for PostgreSQL Website
 title: "[Enhancement]: "
 labels: ["enhancement", "untriaged"]
 assignees:
-  - amazon-auto
+  - 3manuek
+  - gopinathpai
+  - xaviersierra
```

Comment:
Please add me as well. Or maybe we should setup some Github teams and assign stuff to them?